Formula/nopoll.rb

```diff
@@ -0,0 +1,34 @@
+class Nopoll < Formula
+  desc "OpenSource C WebSocket toolkit"
+  homepage "https://www.aspl.es/nopoll/"
+  url "https://www.aspl.es/nopoll/downloads/nopoll-0.4.5.b375.tar.gz"
+  sha256 "4ecbd277714c9acd154277a46388689f9d651836ca5c69990e35d1eee8c7ceae"
+  head "https://github.com/asples/nopoll.git"
+
+  depends_on "openssl"
+
+  depends_on "autoconf" => :build
```

Comment:
Are you sure these are required for the stable build? Looking at the stable tarball it seems unlikely.

Reply:
True, that's a leftover from trying to build HEAD. I will remove it.